shared/macros/fixtext.jinja

```diff
@@ -93,14 +101,66 @@ The audit daemon must be restarted for the changes to take effect.
 {{% endmacro %}}
 
 {{#
-    Generate a fixtext for audit rules for unsuccessful audit - creat
+    Generate a fixtext for audit rules for unsuccessful file modification
+    Used in rules using template audit_rules_unsuccessful_file_modification
+
+:param syscall: Sycall name
+:type path: str
+:param syscall_grouping: A list of other syscalls that can be audited with the
+    same audit rules.
+:type path: str
+
+#}}
+{{% macro fixtext_audit_rules_unsuccessful_file_modification(syscall, syscall_grouping) -%}}
+{{{ fixtext_audit_rules(syscall, "access", syscall_grouping|join(", "), extra_params=True) }}}
```

Comment:
Unfortunately this is not the same as previously rendered text. Previous it has the commas but the last item had `and` as separator. And there all the elements will be separated with a comma. It's probably not a big deal but since it's a text document people expect proper English there I guess.